jekyll/_cci2/notifications.md

```diff
@@ -119,42 +118,38 @@ workflows:
           context: slack-secrets
 ```
 
-CircleCI's Slack orb can be used to provide other types of notifications, including notifying a slack channel of a pending approval or sending a status alert at the end of a job based on success or failure. For more information and to view usage examples, see the [CircleCI Slack orb page](https://circleci.com/developer/orbs/orb/circleci/slack).
+See [Using the Slack Orb]({{ site.baseurl }}/2.0/slack-orb-tutorial/) for a full tutorial with examples.
 
-### Using the IRC Orb
+CircleCI's Slack orb can be used for other types of notification, including notifying a Slack channel of a pending approval. For more information and to view usage examples, see the [CircleCI Slack orb page](https://circleci.com/developer/orbs/orb/circleci/slack).
+
+### Using the IRC orb
 {: #using-the-irc-orb }
 
 The [IRC orb](https://circleci.com/developer/orbs/orb/circleci/irc) is similar to the Slack orb, but only has one main feature: sending custom IRC notifications from CircleCI. Consider this example configuration:
 
 ```yaml
 version: 2.1
+orbs:
+  irc: circleci/irc@0.2.0
 jobs:
   build:
     docker:
-      - image: <docker image>
-        auth:
-          username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+      - image: <Docker image>
     steps:
+      - checkout
       - irc/notify:
-          server: 'IRC-server-to-connect-to' # default: IRC_SERVER environment varible.
-          port: '6667' # default: 6667 if left blank.
-          channel: 'the irc server to post in' # required parameter
-          nick: 'Your IRC nick name' # default: `circleci-bot`
-          message: webhook # default: "Your CircleCI Job has completed."
-orbs:
-  irc: circleci/irc@x.y.z
-version: 2.1
+          server: <IRC-server-to-connect-to> # default: IRC_SERVER environment varible.
+          port: <6667> # default: 6667 if left blank.
+          channel: <the IRC server to post in> # required parameter
+          nick: <Your IRC nickname> # default: `circleci-bot`
+          message: <Build complete!> # default: "Your CircleCI Job has completed."
 workflows:
   your-workflow:
     jobs:
       - build
 ```
+Replace the values in brackets (<>) with your own details.
 
-## Third party tools
-{: #third-party-tools }
-
-### Chroma feedback
-{: #chroma-feedback }
+## See also
 
-[Chroma Feedback](https://github.com/redaxmedia/chroma-feedback) is a command line tool in Python to turn your RGB powered hardware into an build indicator. The idea of such extreme visibility is to encourage developers to instantly repair their builds.
+- [Orbs Introduction]({{ site.baseurl }}/2.0/orb-intro/)
```

Comment:
Maybe we could add a link to the slack orb tutorial tot his see also section too?